cdap-api/src/main/java/io/cdap/cdap/api/lineage/field/EndPoint.java

```diff
@@ -29,7 +29,9 @@
  * program runs.
  */
 public class EndPoint {
+  @Nullable
```

Comment:
It is better to comment on when this will be null

Reply:
Added this as part of the getter